OWML.Common/Menus/IModButton.cs

```diff
@@ -1,6 +1,8 @@
 ﻿using System;
 using UnityEngine.UI;
 
+#pragma warning disable 1591
```

Comment:
Disabling warnings? Not a fan ;)

Reply:
The warnings are almost always correct, and reminds us to clean up our crap ^_^
If we ignore them, we'll never. Now? Maybe :P